src/core/angularfire2.ts

```diff
@@ -12,7 +12,7 @@ function noop() { }
  * Schedules tasks so that they are invoked inside the Zone that is passed in the constructor.
  */
 export class ZoneScheduler implements SchedulerLike {
-  constructor(private zone: any, private delegate = asyncScheduler) { }
+  constructor(private zone: any, private delegate: any = asyncScheduler) { }
```

Comment:
FYI I've been leaving comments for anything I can drop in the next major, mostly old typescript work arounds.

Reply:
Also feel free to suggest any API breaks that would make maintainability easier too, as you saw I already had two different styles of zone patching I was supporting.